src/slash-bedrock/share/brl-fetch/distros/chimera

```diff
@@ -0,0 +1,75 @@
+#!/bedrock/libexec/busybox sh
+#
+# Chimera Linux bootstrap support
+#
+#      This program is free software; you can redistribute it and/or
+#      modify it under the terms of the GNU General Public License
+#      version 2 as published by the Free Software Foundation.
+#
+# Copyright (c) 2016-2019 Daniel Thau <danthau@bedrocklinux.org>
+#
+
+# shellcheck source=src/slash-bedrock/libexec/brl-fetch
+. /bedrock/share/common-code
+trap 'fetch_abort "Unexpected error occurred."' EXIT
+
+check_supported() {
+	false
+}
+
+speed_test_url() {
+	echo "current/main/${distro_arch}/APKINDEX.tar.gz"
```

Comment:
If @wezm doesn't mind https://www.wezm.net/v2/posts/2024/australian-chimera-linux-mirror/ could also be handled here even though it's not officially in any docs yet :)

Reply:
Fine with me :-)